cmd/maya-apiserver/app/server/snapshot_endpoint.go

```diff
@@ -25,6 +25,8 @@ func (s *HTTPServer) snapshotSpecificRequest(resp http.ResponseWriter, req *http
 		return s.snapshotCreate(resp, req)
 	case strings.Contains(path, "/revert/"):
 		return s.snapshotRevert(resp, req)
+	case strings.Contains(path, "/clone/"):
```

Comment:
Do you think it will be good if we explain somewhere in the comments or commit or PR on:
- Snapshot
- Clone
- Promote
This helps contributors as well as docs.

Reply:
Workflow is changed now, API no longer exists.